LayoutTests/fast/dom/getElementsByClassName/ASCII-case-insensitive-expected.txt

```diff
@@ -1,9 +1,8 @@
-FAIL getByClassName('a') should be a A. Was a.
```

Comment:
This test is wrong and needs to be updated. 

Please do not update WebKit to try and make this test pass because then we will no longer match the specification or other browsers. As such, this PR is not correct. Please fix this test instead.

Reply:
If you run this test in Firefox and Chrome, you'll see that they are failing both checks as well. So if we update WebKit to pass this test, we will no longer match them and have a web-compat issue.